src/app/_services/auth.service.ts

```diff
@@ -392,7 +392,17 @@ export class AuthService implements CanActivate {
     let userHasRole = user.hasOwnProperty("role") && user.role
     let userHasBeenPrompted = user.hasOwnProperty("promptedForRole") && user.promptedForRole
     return user && user.isLoggedIn && !userHasRole && !userHasBeenPrompted
-}
+  }
+
+  /**
+   * Determines whether to display an aji intercept to the user
+   * @returns boolean showAJIIntercept
```

Comment:
will be updating this logic once we know who we want to display the intercept to.